package.json

```
@@ -5,7 +5,7 @@
     "main": "index.js",
     "scripts": {
         "pretest": "node node_modules/rimraf/rimraf coverage/* reports/*",
```

It turns out that this rimraf line silently does nothing. Execute rimraf/bin.js instead

---

Good catch.  Fixed.